.github/workflows/nightly-windows.yml

```diff
@@ -16,26 +16,20 @@ jobs:
       fail-fast: false
       matrix:
         vc_boost:
-          - name: msvc-2019_boost_1730
-            image: 'windows-2019'
-            boost_url: 'https://archives.boost.io/release/1.73.0/source/boost_1_73_0.tar.gz'
-            boost_archive_name: 'boost_1_73_0.tar.gz'
-            boost_folder_name: 'boost_1_73_0'
-            boost_include_folder: 'C:\Boost\include\boost-1_73'
-          - name: msvc-2019_boost_1800
-            image: 'windows-2019'
+          - name: msvc-2022_boost_1800
```

Comment:
Is boost 1.78 the minimum version of boost supported on at msvc-2025?

We need a decision on minimum supported boost version. We need to re-evaluate our original min version support decision. We also need to evaluate the min language level to support, C++ 14? Based on these decisions, yes, we should et up to data the versions.

Reply:
i create the linked issue to study this and update the min supported windows and VC compiler versions properly. We need to be sure we do not break any existing users.